docs: added reference to airbnb's article on linaria (#1228)
callstack/linaria: README.md

**[Why use Linaria](/docs/BENEFITS.md)**

> **[Learn how Linaria, Airbnb’s newest choice for web styling, improved both developer experience and web performance](https://medium.com/airbnb-engineering/airbnbs-trip-to-linaria-dc169230bd12)**

Reviewer: ```suggestion
**[Learn how Airbnb improved both developer experience and web performance with Linaria](https://medium.com/airbnb-engineering/airbnbs-trip-to-linaria-dc169230bd12)**
```
